indico/modules/rb_new/client/js/components/pages/BookRoom.jsx

```diff
@@ -64,12 +64,7 @@ export default class BookRoom extends React.Component {
         suggestions: PropTypes.object.isRequired
     };
 
-    constructor(props) {
-        super(props);
-        this.state = {
-            view: 'book'
-        };
-    }
+    state = {};
```

Comment:
can be omitted, especially if there's no local state

Reply:
There is local state.